maintainers/maintainer-list.nix

```diff
@@ -14066,4 +14066,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  asimpson = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.